### DISCUSSION TITLE
Confluent.Kafka.ConsumeException: Fetch from broker 0 failed: Unknown broker error

### DISCUSSION THREAD

**Discussion by ksdvishnukumarcse** [edenhill/librdkafka#4037] [Q&A]
For some reason kafka consumer is throwing the below error while calling the consume method.
"Confluent.Kafka.ConsumeException: Fetch from broker 0 failed: Unknown broker error"

Am using the Confluent Kafka Package with version 1.8.2 to connect to Eventhub.
Below are the consumer config for reference.

BootstrapServers = <EH-NAME-SPACE>.servicebus.windows.net:9093,
SaslUsername = "$ConnectionString",
SaslPassword = <EH-SHARED_ACCESS-POLICY-PRIMARY-CONNETCION-STRING>,
SaslMechanism = SaslMechanism.Plain,
SecurityProtocol = SecurityProtocol.SaslSsl
EnableAutoCommit= true,
PollingTimeoutInMs = 10000,
AutoOffsetReset = AutoOffsetReset.Earliest,
AutoCommitIntervalMs = 500
EnablePartitionEof = true,
IsolationLevel = IsolationLevel.ReadUncommitted,
SocketNagleDisable = true

The above error is happening while running the consumer in AKS. The same is not happening while running the executable locally on my machine.

Any help much appreciated.

**Comment by edenhill** [comment]
Unknown broker errors are broker-side errors, not something the client knows anything about. Check with your Kafka service provider.

**Comment by ksdvishnukumarcse** [comment]
Thanks for the quick reply @edenhill. Definitely will check from service provider.
